src/components/Integrations/BigQuery/FreeAccount/FreeAccount.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+
+import FreeAccount from './FreeAccount.tsx';
+
+describe('Bigquery FreeAccount component', () => {
+  afterEach(cleanup);
+
+  it('renders bigquery free account button', () => {
+    const { getByText } = render(
+      <IntlProvider>
+        <FreeAccount />
+      </IntlProvider>,
+    );
+    expect(getByText('big_query.free_text_summary')).toBeInTheDocument();
+    expect(getByText('big_query.free_btn_redirect')).toBeInTheDocument();
+    expect(getByText('big_query.free_ul_item_one')).toBeInTheDocument();
+    expect(getByText('big_query.free_ul_item_two')).toBeInTheDocument();
+    expect(getByText('big_query.free_title')).toBeInTheDocument();
+    expect(getByText('big_query.free_text_strong')).toBeInTheDocument();
+  });
```

Comment:
Hay forma de testear también el ruteo?

Reply:
In here we have some routing testing https://github.com/FromDoppler/doppler-webapp/blob/master/src/App.test.js#L100 Perhaps you could add a test here.

Reply:
A few time ago, I have added some tests using Route to an specific component: https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/Reports/Subscribers/Subscribers.test.js
Maybe this could help too.